tests/parsers/test_directive_parser.py

```diff
@@ -16,22 +16,22 @@ def test_define(self):
         node = preprocessor.DirectiveParser(tokens).parse()
         self.assertTrue(isinstance(node, preprocessor.DefineNode))
         self.assertTrue(str(node.identifier) == "FOO")
-        self.assertTrue(node.args is None)
+        self.assertTrue(node.argv is None)
```

Comment:
I think these will need to be changed as a result of reverting `argv` to `args` in the classes themselves.